cypress/e2e/DisposableEmailDetectionAPI/documentation.cy.js

```diff
@@ -0,0 +1,166 @@
+import { DISPOSABLE_EMAIL_API } from '../../resources/data';
+
+describe('Disposable Email Detection API Documentation page', () => {
+    before(() => {
+        cy.visit(DISPOSABLE_EMAIL_API);
+    });
+
+    it('1. Test to "Accept Cookies" if not already accepted', () => {
+        // Check if the "Accept Cookies" button is visible or exists
+        cy.AcceptCookies();
+    });
+
+    it('2. Test to check "Documentation" tab visibility and Test to click "Documentation" tab', () => {
+        cy.componentVisiblityCheck('span', 'Documentation');
+        cy.intercept('GET', '/marketplace/disposable_email-api/tabs/api_docs').as('apiRequest');
+        cy.contains('span', 'Documentation').click();
+        cy.wait('@apiRequest');
+    });
+
+    it('3. Test to check "Disposable Email Detection API Reference" title is present', () => {
+        cy.checkTextVisibility('Disposable Email Detection API Reference');
+    });
+
+    it('4. Test to check "development quickstart guide" link visibility and click behavior', () => {
+        cy.componentVisiblityCheck('a[href="/docs/article/getting-started"]', 'development quickstart guide.');
```

Comment:
Can you please add this url to a variable and reuse it?

Reply:
done